documentation/assemblies/assembly-getting-started-strimzi-admin.adoc

```diff
@@ -6,8 +6,13 @@
 = Strimzi Administrators
 
 {ProductName} includes several custom resources.
```

Comment:
_Strimzi provides custom resources for configuration of your deployment._

I moved and did some minor edits to this file for the deployment guide PR.
Will need to make sure the changes in this PR are reflected.